docs/use_cases/model_applications/cryosphere/README.rst

```diff
@@ -1,2 +1,3 @@
 Cryosphere
 ----------
+The frozen part of the earth system modeling framework. Also referred to as sea-ice.
```

Comment:
The frozen part of the earth system modeling framework (also referred to as sea-ice).